src/components/TabsBar.vue

```diff
@@ -167,8 +167,13 @@ export default {
       this.localOpenedPages.splice(this.localOpenedPages.indexOf(pageId), 1);
       this.$emit("tab-closed", this.pages[pageId], this.localOpenedPages);
     },
+    updateTabsReferences(pageDelete) {
+      this.localOpenedPages = this.localOpenedPages.map((page) => {
+        return page > pageDelete ? page - 1 : page;
+      });
```

Comment:
Use arrow function shorthand
```suggestion
      this.localOpenedPages = this.localOpenedPages.map((page) => page > pageDelete ? page - 1 : page);
```